### ISSUE TITLE
[Accessibility] Inability to Interact with "Status Filter" under map view for Keyboard-Only Users 

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4966]
## Describe the bug

* The status filter control in the "Map View" is not keyboard-accessible. Users are unable to navigate and select the status filter checkboxes using only the keyboard, preventing them from filtering the map view using the keyboard.

## To Reproduce

1. Launch the local host
2. Navigate to the "Map View"
3. Locate the status filters under the Map View and try to interact with it without using the mouse.
4. Observe if you are able to complete these actions using the keyboard only.

## Expected behavior

Status filter should be keyboard-focusable and operable, allowing users to interact with it

## Screenshots

![Screenshot from 2024-06-18 18-02-50](https://uploads.linear.app/f1df6e34-3bca-4ade-bb48-e6a457e64091/9f18c9d9-85b8-4496-9eee-813659c759ff/cbdb2dac-d172-4974-b2ea-6fd88f91271f?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2YxZGY2ZTM0LTNiY2EtNGFkZS1iYjQ4LWU2YTQ1N2U2NDA5MS85ZjE4YzlkOS04NWI4LTQ0OTYtOWVlZS04MTM2NTljNzU5ZmYvY2JkYjJkYWMtZDE3Mi00OTc0LWIyZWEtNmZkODhmOTEyNzFmIiwiaWF0IjoxNzE4OTYzMzc1LCJleHAiOjMzMjg5NTIzMzc1fQ.fOzxwb3bqCww11kaQ_1hgmDa7Kgo2eS94m_D5pRVVaM)

**Comment by Angamanga** [comment]
Closing but keeping in Linear: https://linear.app/ushahidi/issue/USH-1436/[accessibility]-inability-to-interact-with-status-filter-under-map
